Foxtrot/Tests/UnitTests/GenericsTest.cs

```diff
@@ -15,286 +15,231 @@
 using System;
 using System.Text;
 using System.Collections.Generic;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
 using System.Diagnostics.Contracts;
 using CodeUnderTest;
+using Xunit;
 
 namespace Tests {
   /// <summary>
   /// Summary description for GenericsTest
   /// </summary>
-  [TestClass]
-  public class GenericsTest : DisableAssertUI {
-    public GenericsTest()
+  public class GenericsTest : DisableAssertUI, IClassFixture<GenericsTest.GenericsTestFixture> {
```

Comment:
:question: Did I mess up the formatting? I tried to match the formatting already in place.